Update monitoring and access control for egress traffic blog post (#3163)
istio/istio.io: content/blog/2018/egress-monitoring-access-control/index.md

 post describes Istio's monitoring and access policies applied exclusively to the egress traffic.

## Before you begin

Follow the steps in the [Egress Gateway with TLS Origination](/docs/examples/advanced-gateways/egress-gateway-tls-origination/) example, **with mutual TLS authentication enabled**, without
the [Cleanup](/docs/examples/advanced-gateways/egress-gateway-tls-origination//#cleanup) step.
After you accomplish this, you will be able
to access [edition.cnn.com/politics](https://edition.cnn.com/politics) from an in-mesh container that has _curl_
installed. In the instructions of this blog post we assume that the `SOURCE_POD` environment variable contains the
source pod's name and the container's name is `sleep`.

## Configure monitoring and access policies

Note that since you want to accomplish your tasks in a _secure way_, you must direct egress traffic through
_egress gateway_, as described in the [Egress Gateway with TLS Origination](/docs/examples/advanced-gateways/egress-gateway-tls-origination/)
task. The _secure way_ here means that you want to prevent malicious applications from bypassing Istio monitoring and
policy enforcement.

> According to the scenario of this blog post, the organization performed the instructions in the

Reviewer: To help readers identify with your message, consider including the reader in the narrative. This technique can be effective to keep the text clear and simple.
```suggestion
According to our scenario, the organization performed the instructions in the
```

Author: @rcaballeromx According to the istio.io style guide (https://istio.io/about/contribute/style-guide/#avoid-using-we), "we" should be avoided, so I assumed that "our" should be avoided too.

> [Before you begin](#before-you-begin) section. It enabled HTTP traffic to _edition.cnn.com_ and configured that traffic

Reviewer: ```suggestion
[Before you begin](#before-you-begin) section, enabled HTTP traffic to _edition.cnn.com_, and configured that traffic
```

to pass through the egress gateway. The egress gateway performs TLS origination to _edition.cnn.com_, so the traffic

> leaves the mesh encrypted. Now the organization is ready to configure Istio for monitoring and access policies for

Reviewer: "Now" is probably not the best word choice. It can confuse readers thinking that you are referring to time but I understand you want to establish the situation before you move on. Consider instead:
```suggestion
leaves the mesh encrypted. At this point, the organization is ready to configure Istio to monitor and apply access policies for
```

Author: Now -> at this point, configure for monitoring -> configure to monitor

---
title: Monitoring and Access Policies for HTTP Egress Traffic
description: Describes how to configure Istio for monitoring and access policies of HTTP egress traffic.

> publishdate: 2018-06-22

Reviewer 2: Please add a new front-matter field:

last_update: 2019-11-29

Author: Done by https://github.com/istio/istio.io/pull/3163/commits/4adf48b4cf57c87e17bb94ab5101ee74119ce0e9. Nice feature!
